### PR TITLE
refactor(plugin): add `isAvailable`

### PR DESCRIPTION
**Pull request by gao-sun**
## Summary

add `isAvailable` for social plugins and filter before passing to webview

## Linear Issue Reference

LOG-2175

## Testing

UT later

## Reviewers

@logto-io/eng

### CODE REVIEW

File: Sources/LogtoSocialPlugin/LogtoSocialPlugin.swift [hunk 1]
```diff
@@ -30,12 +30,17 @@ public struct LogtoSocialPluginConfiguration {
...
     var urlSchemes: [String] { get }
+    var isAvailable: Bool { get }
 
     func handle(url: URL) -> Bool
     func start(_ configuration: LogtoSocialPluginConfiguration)
 }
 
 public extension LogtoSocialPlugin {
+    var isAvailable: Bool {
+        true
```
Comment (IceHe) [inline Sources/LogtoSocialPlugin/LogtoSocialPlugin.swift:41]: should be `true`? maybe `false`?

Comment (gao-sun) [reply Sources/LogtoSocialPlugin/LogtoSocialPlugin.swift:41]: default should be true

Comment (gao-sun) [reply Sources/LogtoSocialPlugin/LogtoSocialPlugin.swift:41]: unless needed, wechat sdk is awful

**Review by simeng-li — APPROVED** [review]
LGTM

File: Sources/LogtoSocialPlugin/LogtoSocialPlugin.swift (hunk 1 above)
Comment (IceHe) [reply Sources/LogtoSocialPlugin/LogtoSocialPlugin.swift:41]: why default true?

Comment (IceHe) [reply Sources/LogtoSocialPlugin/LogtoSocialPlugin.swift:41]: See Slack [thread](https://silverhand-io.slack.com/archives/C02A8G4HVAM/p1649907465208309?thread_ts=1649906599.502909&cid=C02A8G4HVAM)

**Review by wangsijie — APPROVED** [review]

**Pull request merged**